addons/point_of_sale/static/tests/tours/product_screen_tour.js

```diff
@@ -660,3 +660,21 @@ registry.category("web_tour.tours").add("test_barcode_search_attributes_preset",
             Chrome.endTour(),
         ].flat(),
 });
+
+registry.category("web_tour.tours").add("test_single_attributes_dont_show", {
```

Comment:
Maybe one nitpicking since you have to resolve conflicts and push anyway.
```suggestion
registry.category("web_tour.tours").add("test_single_attributes_dont_show_in name", {
```
Might be a bit more explicit. Same for python test